timemachine/cpp/src/nonbonded_interaction_group.cu

```diff
@@ -132,25 +132,36 @@ NonbondedInteractionGroup<RealType, Interpolated>::NonbondedInteractionGroup(
     gpuErrchk(cudaMalloc(&d_sort_vals_in_, N_ * sizeof(d_sort_vals_in_)));
 
     // initialize hilbert curve
-    std::vector<unsigned int> bin_to_idx(256 * 256 * 256);
-    for (int i = 0; i < 256; i++) {
-        for (int j = 0; j < 256; j++) {
-            for (int k = 0; k < 256; k++) {
+    const int N_BINS = 128;
+    const int N_BITS = 8;
+    const int capacity = 1 << N_BITS;
```

Comment:
if moving `N_BINS` to `k_nonbonded.cuh` is an option, seems like these constants could be moved there too?

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/692/commits/afc84f08cbe01169d63f257de12730843800a33a ! Also converted the checks from runtime errors to static_asserts, which seems more appropriate